backend/de.metas.payment.sepa/base/src/main/sql/postgresql/ddl/views/C_Invoice_PaySelectionTrxType_V.sql

```diff
@@ -4,8 +4,8 @@ CREATE OR REPLACE VIEW C_Invoice_PaySelectionTrxType_V AS
 SELECT C_Invoice_ID,
 	CASE 
 		WHEN /* CREDIT_TRANSFER_TO_VENDOR - "OUT" */ i.IsSOTrx='N' AND i.PaymentRule IN ('T'/*DirectDeposit*/, 'P'/*OnCredit*/) AND i.DocStatus IN ('CO', 'CL') THEN 'CT'
-		WHEN /* DIRECT_DEBIT_FROM_CUSTOMER - "CDD" */ i.IsSOTrx='Y' AND dt.DocBaseType = 'ARC' AND i.PaymentRule IN ('D'/*DirectDebit*/) AND i.DocStatus IN ('CO', 'CL') THEN 'DD'
-		WHEN /* CREDIT_TRANSFER_TO_CUSTOMER - "CRE" */ i.IsSOTrx='Y' AND dt.DocBaseType = 'ARC' AND i.PaymentRule IN ('T'/*DirectDeposit*/) AND i.DocStatus IN ('CO', 'CL') THEN 'CT'
+		WHEN /* DIRECT_DEBIT_FROM_CUSTOMER - "CDD" */ i.IsSOTrx='Y' AND dt.DocBaseType != 'ARC' AND i.PaymentRule IN ('D'/*DirectDebit*/, 'P'/*OnCredit*/) AND i.DocStatus IN ('CO', 'CL') THEN 'DD'
```

Comment:
pls change `!='ARC'` to `='ARI'`.

Reason: avoid future doc base types to be matched by this rule.

Reply:
Done